cmd/kube-proxy/app/server.go

```diff
@@ -104,6 +106,8 @@ type proxyRun interface {
 type Options struct {
 	// ConfigFile is the location of the proxy server's configuration file.
 	ConfigFile string
+	// ConfigFileMd5sum is the md5sum of the proxy server's configuration file's content.
+	ConfigFileMd5sum string
```

Comment:
one doubt, should this be a private variable?

Reply:
Good catch! I have made it private.